src/header_dialog.js

```diff
@@ -70,6 +70,7 @@ export function HeaderDialog(dialog, onSave) {
         {name:'dterm_filter_type'            , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'3.0.0', max:'999.9.9'},
         {name:'dterm_filter2_type'           , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'3.0.0', max:'999.9.9'},
         {name:'pidAtMinThrottle'             , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'3.0.0', max:'999.9.9'},
+        {name:'thrust_linear'                , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'3.0.0', max:'999.9.9'},
```

Comment:
```suggestion
        {name:'thrust_linear'                , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.5.0', max:'999.9.9'},
```
min will need to be 4.5.0
https://github.com/betaflight/betaflight/pull/12337